test/karma.conf.profile.js

```diff
@@ -0,0 +1,174 @@
+const path = require('path');
+const { chromium } = require('playwright');
+const webpack = require('webpack');
+
+const workspaceRoot = path.resolve(__dirname, '../');
+
+const browserStack = {
+  username: process.env.BROWSERSTACK_USERNAME,
+  accessKey: process.env.BROWSERSTACK_ACCESS_KEY,
+  build: `material-ui-${new Date().toISOString()}`,
+};
+
+process.env.CHROME_BIN = chromium.executablePath();
+
+// Karma configuration
+module.exports = function setKarmaConfig(config) {
```

Comment:
Is there a way we could share more of the configuration with the regular run karma config?

Reply:
Yes. But premature abstraction is an anti-pattern.